velodrome/transform/fetcher.go

```diff
@@ -26,103 +27,108 @@ import (
 )
 
 // fetchRecentIssues retrieves issues from DB, but only fetches issues modified since last call
-func fetchRecentIssues(db *gorm.DB, last time.Time, out chan sql.Issue) (time.Time, error) {
-	glog.Infof("Fetching issues updated after %s", last)
+func fetchRecentIssues(db *gorm.DB, last *time.Time, out chan sql.Issue) error {
+	glog.Infof("Fetching issues updated after %s", *last)
 
 	var issues []sql.Issue
 	query := db.Where("issue_updated_at >= ?", last).Order("issue_updated_at").Preload("Labels").Find(&issues)
 	if query.Error != nil {
-		return last, query.Error
+		return query.Error
 	}
 
 	count := len(issues)
 	for _, issue := range issues {
 		out <- issue
-		last = issue.IssueUpdatedAt
+		*last = issue.IssueUpdatedAt
```

Comment:
I thought the normal golang thing was to say `last = &issue.IssueUpdatedAt`. Might be wrong.